btr-web/btr-main-app/components/individual-person/tax-info/TaxNumber.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div class="flex flex-col py-5">
+    <div class="flex items-center mb-2 py-1">
+      <URadio
+        id="taxNumberRadioButton"
+        v-model="selectedButton"
+        :value="`CRA Tax Number`"
```

Comment:
It would be good that values are not strings with spaces. Perfect would be Enums, but sometimes they might not work, so string values are OK, but they should be named more similar to how variables are named.
One word in CamelCase.